spec/controllers/institutions_controller_spec.rb

```diff
@@ -63,4 +108,32 @@
 
   end
 
+  describe "new_from_invite_data" do
+    it "with invite" do
+      invite = PendingInstitutionInvite.make(invited_user_email: user.email)
+      get :new_from_invite_data, {pending_institution_invite_id: invite.id}
+      expect(response).to be_success
+    end
+
+    it "without invite" do
+      expect do
+        get :new_from_invite_data
+      end.to raise_error(ActionView::MissingTemplate)
+    end
+
+    it "with accepted invite" do
+      invite = PendingInstitutionInvite.make(status: "accepted", invited_user_email: user.email)
+      expect do
+        get :new_from_invite_data, {pending_institution_invite_id: invite.id}
+      end.to raise_error(ActionView::MissingTemplate)
+    end
```

Comment:
I think this is also a bug: shouldn't we report a message that the invitation has already been accepted, instead of crashing? @gbatiston @diegoliberman

Reply:
Yes, that would be better.

Reply:
This is tracked in #1467